### ISSUE TITLE
Consul command not available on Windows

### ISSUE THREAD

**Issue by emachnic** [sous-chefs/consul#451]
Just a quick fix but currently, the Windows installation does not add the Consul directory to the path and trying to run `consul` commands from Powershell do not work. I was able to get around this by using `windows_path` from the Windows cookbook like so:

```ruby
windows_path "C:/Program Files/consul/#{node['consul']['version']}" do
  action :add
end
```

It might be cool to go ahead and do this in the cookbook already so that the consumers of the cookbook don't have to.

Cheers,
Evan

**Comment by gdavison** [comment]
A shortcut might be a better option, in the same way that the *nix versions use a symlink. I haven't tested that this works, though

**Comment by gdavison** [comment]
@emachnic, a Windows symlink will do it. I have code on https://github.com/paybyphone/consul-cookbook/tree/windows_executable_symlink that creates it.

It's waiting on #452 to fix Windows issues with cookbook version 3.0

**Comment by gdavison** [comment]
@emachnic This is available on the `master` branch, but not yet in a numbered release

**Comment by damacus** [comment]
Closing due to inactivity. 

If this is still an issue please reopen or open another issue. Alternatively drop by the #sous-chefs channel on the [Chef Community Slack](http://community-slack.chef.io/) and we'll be happy to help! 

Thanks,
Sous-Chefs
